app.rb

```diff
@@ -0,0 +1,134 @@
+require './students'
```

Comment:
[ ] I get the error below when I run the project because your file is named `student `and not students. kindly fix that 🤓. except that everything is working perfectly fix it and send it for rereview.
![errors](https://github.com/Young-Gaius/School-Library/assets/62156607/11463707-c07d-4071-a313-16a264c54d5b)